utils/githooks/install

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -eu
+
+githook_root="$(dirname $0)"
```

Comment:
```suggestion
githook_root="${0%/*)"
```

Reply:
I don't understand this syntax, and it's not valid:
./utils/githooks/install: line 14: unexpected EOF while looking for matching `"'

Reply:
Actually, my eyes weren't working... It just needs to be
githook_root="${0%/*}"

Thanks!